source/jormungandr/jormungandr/scenarios/default.py

```diff
@@ -167,6 +168,7 @@ def parse_journey_request(self, requested_type, request):
         return req
 
     def call_kraken(self, req, instance, tag=None):
+        record_custom_parameter('scenario', "simple")
```

Comment:
minor (same below): consistency about the simple quote would be nice if possible, so `'simple'`